halo2_proofs/src/arithmetic.rs

```diff
@@ -166,29 +166,11 @@ pub fn best_multiexp<C: CurveAffine>(coeffs: &[C::Scalar], bases: &[C]) -> C::Cu
 /// distinct powers of $\omega$. This transformation is invertible by providing
 /// $\omega^{-1}$ in place of $\omega$ and dividing each resulting field element
 /// by $n$.
-///
-/// This will use multithreading if beneficial.
```

Comment:
It's still the case that this will use multithreading if beneficial, so revert this change.

Reply:
Thank you.
I modified.
Do you mean we use spawn?
https://doc.rust-lang.org/book/ch16-01-threads.html#creating-a-new-thread-with-spawn